www/source/layouts/_footer.slim

```diff
@@ -1,111 +1,53 @@
-                h4.footer--subheader Support
-              li.footer--sitemap--link
-                a href="https://github.com/habitat-sh/habitat/issues" New Issues
-              li.footer--sitemap--link
-                a href="https://portal.prodpad.com/24539" Feature Request
-              li.footer--sitemap--link
-                a href="http://slack.habitat.sh/" Habitat Slack
-              li.footer--sitemap--link
-                a href="https://status.habitat.sh/" target="_blank" Status
-
-          .columns.medium-2.small-6
-            ul.no-bullet.social
-              li
-                h4.footer--subheader Around the Web
-              li.footer--sitemap--link
-                a.github href="https://github.com/habitat-sh/habitat" GitHub
-              li.footer--sitemap--link
-                a.twitter href="https://twitter.com/habitatsh" Twitter
-              li.footer--sitemap--link
-                a.youtube href="https://www.youtube.com/playlist?list=PL11cZfNdwNyOxlvI1Kq6ae8eVBl5S3IKk" YouTube
-
-        .row
-          .columns.medium-12
-            ul.no-bullet.footer--copyright
-              li.footer--sitemap--link
-                p &copy; #{Time.now.year} Chef Software, Inc. All Rights Reserved. Patent Pending.
-              li.footer--sitemap--link
-                a href="/legal/licensing" Licensing
-              li.footer--sitemap--link
-                a href="/legal/terms-and-conditions" Terms &amp; Conditions
-              li.footer--sitemap--link
-                a href="/legal/trademark-policy" Trademark Policy
-              li.footer--sitemap--link
-                a href="/legal/privacy-policy" Privacy Policy
-              li.footer--sitemap--link
-                a href="https://github.com/habitat-sh/habitat/blob/master/CONTRIBUTING.md#contributing-to-habitat" Contributing
-              li.footer--sitemap--link
-                a href="https://github.com/habitat-sh/habitat/blob/master/CODE_OF_CONDUCT.md" Code of Conduct
-              li.footer--sitemap--link
-                a href="https://www.chef.io/cookie-policy/" Cookie Policy
+  .row.graphics
+    .columns.small-offset-8.small-4.end
+      img.footer--habicat src='/images/graphics/habicat-wave.svg'
+  .row.footer--sitemap
+    .columns.large-2.small-12
+      a.footer--logo.habitat href="/" title="Habitat Home"
+        img[src="/images/habitat-logo-by-chef-white.svg"
+          onerror="this.src='/images/habitat-logo-by-chef-white.png'" alt="Habitat by Chef"]
+    .columns.large-8.small-12.center
+      ul.footer--links.inline
+        li.footer--sitemap--link
+          =link_to 'Learn', '/learn'
+        li.footer--sitemap--link
+          =link_to 'Docs', '/docs'
+        li.footer--sitemap--link
+          =link_to 'Community', '/community'
+        li.footer--sitemap--link
+          =link_to 'Enterprise', '/enterprise'
+        li.footer--sitemap--link
+          =link_to 'Blog', '/blog'
+        li.footer--sitemap--link
```

Comment:
This ends up rendering as an empty `a` tag, which throws off the center alignment of the footer items on mobile:

![image](https://user-images.githubusercontent.com/274700/38900509-7f9af7ee-4250-11e8-8bd4-8aeba3e9acd8.png)

Do we need a link to Builder in the footer? If we do, we should probably put the word `Builder` in there.

Reply:
done!